charts/seqr/values.yaml

```diff
@@ -99,6 +99,12 @@ cronJobs:
   - name: check-new-samples-job
     schedule: "*/10 * * * *"
     command: "python manage.py check_for_new_samples_from_pipeline"
+  - name: update-all-reference-data-job
+    schedule: "0 4 * * 0"  # 4am on sunday
+    command: "python manage.py update_all_reference_data --use-cached-omim"
```

Comment:
This is not a good idea, it deletes all of genocode and would functionally cause like a half hour of degraded performance and incorrect searches. This job re we do not even run this as a cron job we runna series of more targeted updates. I will do a post MVP audit of our cron jobs and adapt the ones that are useful for local insta

Reply:
ah ok!  I had it written down that we should do this but I must have misunderstood.

Reply:
yeah sorry I was musing that we should do it in the future and I guess I was not being clear